unix-alpha/Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 pluginbase = "*"
-ansible = "==5.5.0"
+ansible = "*"
```

Comment:
Silly question, Is it a good idea to keep this `"*"`? why not match it to the actual version `5.6.0`

Reply:
Now that we are sure that the Ansible version 5 is working as expected and doesn't require any changes in the code, it is fine to keep this to `*`. We will still have Ansible locked to 5.6 from the `Pipfile.lock` and the file `setup.py` also mentions on line#19 that we require ansible 5.6 installed for machine stats.